### ISSUE TITLE
`PeriodicContainer` issue with multiple branches

### ISSUE THREAD

**Issue by eleonoracastelli** [mikekatz04/Eryn#14]
Submitting this after bothering @mikekatz04 offline (sorry Michael).

Michael When running an RJ mcmc with multiple branches with different `ndims`, if only one branch has a cyclic parameter declared via `periodic = PeriodicContainer(periodic_in)` a `ValueError` is triggered.

Here are the details of my minimal example:

```python
branch_names = ['cyclic', 'notcyclic']
# priors
priors = {"cyclic": ProbDistContainer({
             0: log_uniform(10**-3, 10**3), # (10**0, 10**2)
             1: log_uniform(0, fmax),
             2: log_uniform(10**-1, 10**2),
             3: uniform_dist(0, 2*np.pi),
             4: uniform_dist(0, tmax),
            }), 
          "notcyclic": ProbDistContainer({
             0: log_uniform(10**-3, 10**3),
             1: log_uniform(10**-4, 10**2),
             2: uniform_dist(0, tmax),
            })}

# which parameters are cyclic
periodic = PeriodicContainer({"cyclic": {
                       2: 2*np.pi
                       }})

nwalkers    = 100
ntemps   = 14

ndim     = {"cyclic": 5,
            "notcyclic": 3}

nleaves_min = {"cyclic": 1,
               "notcyclic": 1}

nleaves_max = {"cyclic": 3,
               "notcyclic": 3}

moves  = GaussianMove(cov)
```

And the triggered error is the following, occurring at line `File "/Users/ecastel2/Documents/virtual-envs/global-venv/lib/python3.10/site-packages/eryn/moves/gaussian.py", line 114, in <dictcomp>  name: tmp.reshape(ntemps * nwalkers, nleaves_max, ndim)`

```python
ValueError: cannot reshape array of size 21000 into shape (1400, 3, 3)
```
where 21000 = 14 * 100 * 5 * 3.

If I take out the cyclic condition, the code runs just fine. Need to run some additional tests to see if I just need to declare the cyclic parameter for the `'noncylic'` branch to be `None`.

**Comment by eleonoracastelli** [comment]
After troubleshooting it together, the solution found by @mikekatz04 is the following: substitute lines `111-123` of `moves/gaussian.py` with

```python
        if self.periodic is not None:
            q = self.periodic.wrap(
                {
                    name: tmp.reshape((ntemps * nwalkers,) + tmp.shape[-2:])
                    for name, tmp in q.items()
                },
                xp=self.xp,
            )

            q = {
                name: tmp.reshape((ntemps, nwalkers,) + tmp.shape[-2:])
                for name, tmp in q.items()
            }
```

**Comment by mikekatz04** [comment]
fixed in version 1.1.7
